test/gtest/ucs/test_config.cc

```diff
@@ -579,6 +586,35 @@ UCS_TEST_F(test_config, set_get) {
     EXPECT_EQ(1, m_num_errors);
 }
 
+UCS_TEST_F(test_config, set_blob)
+{
+    static const char ucs_cars_prefix[] = UCS_DEFAULT_ENV_PREFIX"CARS_";
+    ucs::scoped_setenv ucx_config_dir("UCX_CONFIG_DIR", TEST_CONFIG_DIR);
+    car_opts mazda(ucs_cars_prefix, "MAZDA_");
+
+    const std::string model2("2"), model3("3"), model6("6"), modelcx5("cx5");
+    ucs_status_t status;
+    // no prefix
+    status = mazda.set("MODEL", model2.c_str());
+    ASSERT_UCS_OK(status);
+    ASSERT_EQ(model2, mazda.get("MODEL"));
+
+    // with exact prefix
+    status = mazda.set("MAZDA_", "MAZDA_MODEL", model3.c_str());
+    ASSERT_UCS_OK(status);
+    ASSERT_EQ(model3, mazda.get("MODEL"));
+
+    // with prefix but ignore by mask
+    status = mazda.set("MAZDA_", "*MODEL", model6.c_str());
+    ASSERT_UCS_OK(status);
+    ASSERT_EQ(model6, mazda.get("MODEL"));
+
+    // without prefix and ignore by mask
+    status = mazda.set("*MODEL", modelcx5.c_str());
```

Comment:
Here we set just a single config field, this is not very relevant coverage for wildcard IMO.
It seems this improvement is all about covering multiple values with single wildcard, right?
So this should be tested, e.g. by extending `car_opts_table` with another table with prefix and checking that wildcard setting is applied for both